<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f4f4f4; /* Fundo claro */
    border: 1px solid #d3d3d3; /* Bordas sutis */
    border-radius: 10px; /* Bordas arredondadas */
}

QMenuBar {
    background-color: #ffffff; /* Fundo do menu */
    border-bottom: 1px solid #d3d3d3; /* Linha separadora */
    font-size: 14px;
}

QMenuBar::item {
    padding: 5px 15px;
    background-color: transparent;
}

QMenuBar::item:selected {
    background-color: #e0e0e0; /* Destaque ao passar o mouse */
}

QStatusBar {
    background-color: #ffffff;
    border-top: 1px solid #d3d3d3;
    font-size: 12px;
    color: #606060;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Local de Armazenamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarLocalArmazenamento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarProduto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Produto - (Fitas ou barcode)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarEstoque">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade em estoque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alterar Cadastro de Fitas e Barcodes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #707070;
	color: rgb(161, 161, 161);
    border-radius: 10px;
    color: white;
    font-size: 16px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #5a52d4;
}</string>
    </property>
    <property name="text">
     <string>Confirmar a Alteração</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota Fiscal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarNf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descrição do item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlterarDescricao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tela de Alteração</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
